mytoyota/models/vehicle.py

```diff
@@ -104,64 +101,79 @@ async def parallel_wrapper(name: str, function: partial) -> Tuple[str, Dict[str,
 
     @property
     def vin(self) -> Optional[str]:
-        """Vehicle's vinnumber."""
+        """
+        Returns the vehicles VIN number
+
+        Returns:
+            The vehicles VIN number
+
+        """
         return self._vehicle_info.vin
 
     @property
     def alias(self) -> Optional[str]:
-        """Vehicle's alias."""
-        return self._vehicle_info.nickname
-
-    async def set_alias(self, value) -> None:
         """
-        Sets the alias for the vehicle.
-
-        Args:
-            value: The alias value to set for the vehicle.
+        Vehicle's alias.
 
         Returns:
-            None
+            Nickname of vehicle
 
         """
-        await self._api.set_vehicle_alias_endpoint(value, self._vehicle_info.subscriber_guid, self.vin)
+        return self._vehicle_info.nickname
 
     @property
-    def hybrid(self) -> bool:
-        """If the vehicle is a hybrid."""
-        # TODO need more details to check of electric cars return different capabilities
-        return self._vehicle_info.ev_vehicle
-
-    # @property
-    # def fueltype(self) -> str:
-    #    """Fuel type of the vehicle."""
-    #    fuel_type = self._vehicle_info.get("fuelType", "Unknown")
-    #    if fuel_type != "Unknown":
-    #        # Need to know further types. Only seen "I" or petrol cars.
-    #        fuel_types = {"I": "Petrol"}
-    #        if fuel_type in fuel_types:
-    #            return fuel_types["fuelType"]
-    #        else:
-    #            logging.warning(f"Unknown fuel type: {fuel_type}")
-    #    return "Unknown"
+    def type(self) -> Optional[str]:
```

Comment:
`type `is a little too unspecific for me in terms of the name. How about `drive_type`?

Reply:
Yes, one of the most difficult problems in IT. Finding the right name 😄 
`vehicle_drive_system` maybe?